python/src/equistore/block.py

```diff
@@ -7,7 +7,7 @@
 from .data import Array, ArrayWrapper, eqs_array_to_python_array
 from .labels import Labels
 from .status import _check_pointer
-
+import numpy as np
```

Comment:
The linter seems not to be soo smart, so it throws an error here
https://github.com/lab-cosmo/equistore/actions/runs/4416932020/jobs/7741914595#step:7:512
In the code you don't need the Labels, so it complains. It apparently cannot see that it is used in the doc.
This should fix the problem
```suggestion
import numpy as np # noqa: F401
```

Reply:
We decided to move it to the doc, so here we remove it
```suggestion

```
(see second suggestion where to add it)